js/src/edit.js

```diff
@@ -29,6 +23,19 @@ if( window.wpseoPostScraperL10n ) {
 	localizedData = wpseoTermScraperL10n;
 }
 
+/**
+ * Registers a redux store to Gutenberg.
+ *
+ * @returns {store} The store.
+ */
+function registerStore() {
```

Comment:
I would suggest changing the name of this function, since having the same name as the `registerStore` function from `wp.data` is slightly confusing, especially since this function in edit.js only gets triggered by Gutenberg data being available